src/couch_file.erl

```diff
@@ -531,27 +537,96 @@ find_header(Fd, Block) ->
     {ok, Bin} ->
         {ok, Bin};
     _Error ->
-        find_header(Fd, Block -1)
+        find_header_pread2(Fd, Block -1)
```

Comment:
Rename find_header_pread2 to find_header and then just call the find_header/3 version directly (and remove find_header_pread2/2). Alsos move that function to below this one so the ordering makes more sense.